person.rb

```diff
@@ -10,6 +11,7 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @name = name
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rentals = []
```

Comment:
- [ ] Before merging, please make sure that you add getter and setter for `@rentals` via attr_accessor. It will help you to fetch @rentals's value and to modify it.